lib/config.js

```diff
@@ -138,15 +138,21 @@ var util = Config.prototype.util = {};
  * @return value {mixed} - Property value, including undefined if not defined.
  */
 var getImpl= function(object, property) {
+  // Check for null to acquiesce tests for checking has() on a null object
+  if (object === null) return undefined;
```

Comment:
Does this have anything to do with submodule /config dirs? If not I'd prefer it be in a separate change list, with a separate test that outlines the problem.